outputs.tf

```diff
@@ -0,0 +1,6 @@
+output "webapp_url" {
+    value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+output "cd_webhook" {
+    value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
You should set `sensitive = true` for the webhook - it contains your deployment password.

This setting will stop it appearing in Travis logs after it does the apply. You should also do the same for sensitive variables, but I don't think they're going to get logged currently - it would stop them appearing in plans etc